crypto/rsa/rsa_oaep.c

```diff
@@ -244,15 +243,16 @@ int RSA_padding_check_PKCS1_OAEP_mgf1(unsigned char *to, int tlen,
      * should be noted that failure is indistinguishable from normal
      * operation if |tlen| is fixed by protocol.
      */
-    tlen = constant_time_select_int(constant_time_lt(dblen, tlen), dblen, tlen);
+    if (tlen > dblen - mdlen - 1)
+        tlen = dblen - mdlen - 1;
```

Comment:
Why is this not constant time?

Reply:
dblen is not a secret, mdlen is not a secret,
and tlen controls the loop below, and is therefore also not secret, right?

Reply:
I agree they are not secret, but this is the kind of code where I would like to see as few branches as possible.

Reply:
Well, okay, I can drop that.